index.html

```diff
@@ -0,0 +1,778 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+      integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="./css/style.css" type="text/css" />
+    <title>TNW</title>
+  </head>
+  <body>
+    <header>
+      <div id="small_nav">
+        <img
+          style="width:30px"
+          src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg"
+          alt="img"
+        />
+        <div class="right_small_nav">
+          <i class="fas fa-search"></i>
+          <i class="fas fa-bars"></i>
+        </div>
+      </div>
+      <div id="top_menu">
+        <ul class="flex">
+          <li>News</li>
+          <li>Events</li>
+          <li>Business</li>
+          <li>AMAs</li>
+          <li>Spaces</li>
+          <li>Terms &amp; Conditions</li>
+        </ul>
+      </div>
+      <nav>
+        <ul class="flex">
+          <li>LATEST</li>
+          <li>HARD FORK</li>
+          <li>PLUGGED</li>
+          <li>THE NEXT POLICE</li>
+          <li>FUNDAMENTALS</li>
+          <li>IOTLAB</li>
+          <li>WORK2030</li>
+        </ul>
+
+        <ul>
+          <li><i class="fas fa-search"></i></li>
+        </ul>
+      </nav>
+
+      <div id="message">
+        TNW Conference starts NOW! Tune in to the livestream to watch
+        <i class="fas fa-arrow-right"></i>
+      </div>
+    </header>
+
+    <main class="main_content">
+      <div class="highlight">
+        <!-- START OF HIGHLIGHT SECTION-->
```

Comment:
Let's use semantic tags for every article in this `highlight` section with `<article>` tag